dev-resources/puppetlabs/services/certificate_authority/certificate_authority_int_test/ca_true_test/master/conf/ssl/ca/requests/test_cert_ca_true.pem

```diff
@@ -0,0 +1 @@
+
```

Comment:
Yeah, I think creating the requests directory from inside the test would be better - you should be able to do something like `(fs/mk-dir <directory name string>)` IIRC (where `me.raynes.fs` is aliased to `fs`).